styles.css

```diff
@@ -218,12 +232,12 @@ body {
 }
 
 .see:hover {
-  background-color: #6070ff;
+  background-color: #5cabcf;
   color: #fff;
   transform: translateY(-3px);
 }
 
-.see:visited {
+.see:active {
   background-color: #2230d2;
   color: #fff;
 }
```

Comment:
- [Optional] Great Job for using hover and active state 👌 however your color contrast is failing here is your [results](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-02-28-at-11-38-52-037?failing) so please update it for better contrast, 👍 you can check using [this checker](https://www.aditus.io/button-contrast-checker/)